src/loaders/loaders.base.js

```diff
@@ -101,50 +110,73 @@ export default class LoadersBase {
     });
   }
 
+  /**
+   * parse the data loaded
+   * SHOULD BE implementd by detail loader.
+   * @param {object} response - loaded data.
+   * @return {promise} promise.
+   */
   parse(response) {
     return new Promise((resolve, reject) => {
-      resolve(null);
+      resolve(response);
     });
   }
 
-  // default load sequence promise
+  /**
+   * default load sequence promise.
+   * @param {string} url - resource url.
+   * @return {promise} promise.
+   */
   loadSequence(url) {
     return this.fetch(url)
       .then((rawdata) => {
         return this.parse(rawdata);
       })
       .then((data) => {
         this._data.push(data);
+        return data;
```

Comment:
Return the data, so we can just get the data by:

``` js
loadSequence(url).then((data) => {
  // do with data
})
```

And with `promise.all`, we can get a array filled with data loaded by single resource url.